src/jsonUtils/style.js

```diff
@@ -46,11 +53,11 @@ export const makeShadow = (style: ViewStyle): SJShadow => {
     _class: 'shadow',
     isEnabled: true,
     blurRadius: radius,
-    color: makeColorFromCSS(color),
+    color: makeColorFromCSS(color, opacity),
     contextSettings: {
       _class: 'graphicsContextSettings',
       blendMode: 0,
-      opacity,
+      opacity: 1,
```

Comment:
so just to be clear, the only real change is this and `makeColorFromCSS`, right?